assets/app/scripts/controllers/pods.js

```diff
@@ -20,8 +20,48 @@ angular.module('openshiftConsole')
     $scope.emptyMessage = "Loading...";
     var watches = [];
 
+    $scope.gotoContainerView = function(container, jolokiaUrl) {
+      // console.log("OPENSHIFT_CONFIG: ", OPENSHIFT_CONFIG);
+      // console.log("Clicked! container: ", container, " jolokiaUrl: ", jolokiaUrl);
+      var returnTo = window.location.href;
+      var title = Core.pathGet(container, ['name']) || 'Untitled Container';
+      var targetURI = new URI();
+      targetURI.path('/console/java/')
```

Comment:
Can't actually use $location here as we actually want to go to a different page and not a different route.  But I can get the base.href value and use that to get the right URL.